src/ducks/Signals/TriggerForm.js

```diff
@@ -82,86 +96,94 @@ const ARGS = {
   price_percent_change: ['percentThreshold', 'timeWindow']
 }
 
-const defaultValues = {
+const PRICE_DEFAULT_VALUES = {
   price_percent_change: {
     cooldown: '24h',
     percentThreshold: 5,
-    target: { value: 'santiment', label: 'santiment' },
+    // target: { value: 'santiment', label: 'santiment' },
+    // metric: { label: 'Price', value: 'price' },
     timeWindow: 24,
     timeWindowUnit: { label: 'hours', value: 'h' },
-    type: { value: 'price_percent_change', label: 'Moving up %' },
-    metric: { label: 'Price', value: 'price' },
+    type: {
+      value: PRICE_PERCENT_CHANGE,
+      label: 'Moving up %',
+      type: PRICE_CHANGE_TYPES.MOVING_UP
+    },
     isRepeating: true,
     channels: ['telegram']
   },
   daily_active_addresses: {
     cooldown: '24h',
     percentThreshold: 200,
-    target: { value: 'santiment', label: 'santiment' },
-    metric: {
-      label: 'Daily Active Addresses',
-      value: 'daily_active_addresses'
-    },
+    // target: { value: 'santiment', label: 'santiment' },
+    // metric: { label: 'Price', value: 'price' },
     timeWindow: 2,
     timeWindowUnit: { label: 'days', value: 'd' },
-    type: { label: 'Daily Active Addresses', value: 'daily_active_addresses' },
+    type: { label: 'Daily Active Addresses', value: DAILY_ACTIVE_ADDRESSES },
     isRepeating: true,
     channels: ['telegram']
   },
   price_volume_difference: {
     cooldown: '24h',
     threshold: 0.002,
-    target: { value: 'santiment', label: 'santiment' },
-    metric: {
-      label: 'Price/volume difference',
-      value: 'price_volume_difference'
-    },
+    // target: { value: 'santiment', label: 'santiment' },
+    // metric: { label: 'Price', value: 'price' },
     type: {
       label: 'Price/volume difference',
-      value: 'price_volume_difference'
+      value: PRICE_VOLUME_DIFFERENCE
     },
     isRepeating: true,
     channels: ['telegram']
   }
 }
 
-const DEFAULT_FORM_SETTINGS = defaultValues['price_percent_change']
-
 const getTypeByMetric = metric => {
   if (metric.value === 'price') {
     return {
-      value: 'price_percent_change',
-      label: 'Moving up %'
+      value: PRICE_PERCENT_CHANGE,
+      label: 'Moving up %',
+      type: PRICE_TYPES.MOVING_UP
     }
   } else {
-    return TYPES[metric.value][0]
+    return PRICE_TYPES[metric.value][0]
   }
 }
 
 const mapValuesToTriggerProps = values => ({
   cooldown: values.cooldown,
   settings: (() => {
+    const type = values.type ? values.type.value : undefined
```

Comment:
will be nice, if you'll rewrite this with destructuring incoming `values` and, after that, you can write shortly and without a lot of repeating: 
`cooldown` instead `cooldown: values.cooldown`,
`target: { slug: value }` instead `target: { slug: values.target.value }`,
and etc.

Reply:
Fixed